sections/footer.liquid

```diff
@@ -76,9 +76,9 @@
                 {%- when 'image' -%}
                   <div class="footer-block__details-content footer-block-image {{ block.settings.alignment }}">
                     {%- if block.settings.image != blank -%}
-                      {%- assign image_size = block.settings.image_width | append: 'x' -%}
+                      {%- assign image_size_2x = block.settings.image_width | times: 2 | at_most: 5760 -%}
                       <img
-                        srcset= "{{ block.settings.image | img_url: image_size }}, {{ block.settings.image | img_url: image_size, scale: 2 }} 2x"
+                        srcset= "{{ block.settings.image | image_url: width: block.settings.image_width }}, {{ block.settings.image | image_url: width: image_size_2x }} 2x"
                         src="{{ block.settings.image | image_url: width: 400 }}"
```

Comment:
@ludoboludo I noticed that in the `header.liquid`, the width of the `src` attribute is set to the width of the image while this one is set to `400`. Any insight on why this was set to `400`? I'm happy to change it to `block.settings.image_width`.

- https://github.com/Shopify/dawn/pull/134/files#diff-0f3f44a79ab31e1d43ddbed9fdc379488cc710598937d73b57db6c9a8d90177bR80-R82